--- hidapi-hidapi-0.9.0/pc/hidapi-hidraw.pc.in	2019-06-19 11:34:38.000000000 +0100
+++ hidapi-hidapi-0.9.0/pc/hidapi-hidraw.pc.in-new	2020-10-13 17:56:28.832196857 +0100
@@ -6,5 +6,5 @@ includedir=@includedir@
 Name: hidapi-hidraw
 Description: C Library for USB/Bluetooth HID device access from Linux, Mac OS X, FreeBSD, and Windows. This is the hidraw implementation.
 Version: @VERSION@
-Libs: -L${libdir} -lhidapi-hidraw
+Libs: -L${libdir} -l:libhidapi-hidraw.a
 Cflags: -I${includedir}/hidapi
